Games/Game

```diff
@@ -0,0 +1,459 @@
+import random
+
+randomNumber = random.randint(1, 15)
+##print(int(randomNumber))
+#""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+var = (int(randomNumber))
+var1 = (int(randomNumber + 1))
+
+var2 = (int(randomNumber + 3))
+var3 = (int(randomNumber + 2))
+
+var4 = (int(randomNumber + 26))
+var5 = (int(randomNumber + 22))
+
+var6 = (int(randomNumber + 3))
+var7 = (int(randomNumber + 6))
+
+var8 = (int(randomNumber * 3))
+var9 = (int(randomNumber + 37))
+
+var10 = (int(randomNumber + 112))
+var11 = (int(randomNumber + 86))
+
+var12 = (int(randomNumber + 4))
+
+var13 = (int(randomNumber * 2))
+
+var14 = (int(randomNumber + 5))
+
+var15 = (int(randomNumber + 18))
+
+var16 = (int(randomNumber + 3))
+
+var17 = (int(randomNumber + 16))
+var18 = (int(randomNumber + 14))
+
+var19 = (int(randomNumber + 17))
+var20 = (int(randomNumber + 23))
+
+var21 = (int(randomNumber + 44))
+var22 = (int(randomNumber + 44))
+
+#""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+x = 0
+score = x
+
+
+#---------------------------------------------------------------------------------------
+#this is for our calculator
+# This function adds two numbers
+def add(x, y):
+    return x + y
+
+
+# This function subtracts two numbers
+def subtract(x, y):
+    return x - y
+
+
+# This function multiplies two numbers
+def multiply(x, y):
+    return x * y
+
+
+# This function divides two numbers
+def divide(x, y):
+    return x / y
+
+
+# Import math Library
+import math
+
+import pi
+# Print the value of pi
+#print (math.pi)
+
```

Comment:
Remember our talk about "functions" and why we use them?  The rest of the code would be way easier to follow and maintain if it would to be broken up into functions and also put into different files because this file is so long.